### ISSUE TITLE
VSCode like Editor ?

### ISSUE THREAD

**Issue by murbanowicz** [ChrisRM/material-theme-jetbrains#877]
Would that be possible to get Material Darker VS Code or something like that?

VS Code colors are just great, but I love WebStorm with Material Theme :)

**Comment by mallowigi** [comment]
I will look at it

**Comment by mallowigi** [comment]
Can you elaborate a bit? What is the issue exactly?

**Comment by murbanowicz** [comment]
@mallowigi  - why it got closed?

**Comment by mallowigi** [comment]
Can you elaborate what is it you need ?

**Comment by murbanowicz** [comment]
Yes. I meant VS Code color style, like we have 'Material Darker', we could
have 'Material Code' or something like that as many people prefer WS but
still love VS Code colors which are great.


pon., 6 sie 2018 o 12:07 Elior Boukhobza <notifications@github.com>
napisał(a):

**Comment by mallowigi** [comment]
The problem is that what’s in Code doesn’t necessary apply to Webstorm and vice versa. Some keywords cannot be highlighted differently, but the color palette is definitely the same.
